crates/red_knot_python_semantic/resources/mdtest/type_properties/is_assignable_to.md

```diff
@@ -476,4 +476,28 @@ static_assert(is_assignable_to(CallableTypeOf[keyword_variadic], Callable[..., N
 static_assert(is_assignable_to(CallableTypeOf[mixed], Callable[..., None]))
 ```
 
+### Non-static types
```

Comment:
I think I'd name the header as "Function types" as the "Callable" section is mostly for non-static / gradual types because the cases with fully static types are included in `is_fully_static` test cases.